src/components/definition-list/definition-list.style.js

```diff
@@ -0,0 +1,70 @@
+import styled, { css } from 'styled-components';
+import { space } from 'styled-system';
+import { baseTheme } from '../../style/themes';
+import StyledButton from '../button/button.style';
+import LinkStyle from '../link/link.style';
+
+export const StyledDl = styled.dl`
+  display: inline-flex;
+  height: auto;
+  width: 100%;
+  background-color: transparent;
+  overflow: hidden;
+`;
+
+export const StyledDtDiv = styled.div`
+${space}
```

Comment:
This just needs indenting by a couple of spaces, same with the ones below

Reply:
✅